### PR TITLE
Updating event's card title and subheader

### PR DESCRIPTION
**Pull request by kamcknig**
Adding date and time to the event in the Card's subheader.

### CODE REVIEW

**Comment by kamcknig** [comment]
@scriptist

File: src/react/page/schedule/ScedulePage.tsx [hunk 1]
```diff
@@ -36,24 +43,49 @@ export function EventCard(_props: EventCardProps) {
...
       />
+      <CardContent>{_props.event.description}</CardContent>
     </Card>
   );
 }
+
+/**
+ * Renders the CardHeader subheader element. The event's start time and date
+ */
+function EventSubHeader(_props: EventCardProps) {
```
Comment (scriptist) [inline src/react/page/schedule/ScedulePage.tsx]: Could you please desctructure the props in all three components in this file?

```suggestion
function EventSubHeader({event}: EventCardProps) {
```

File: docs/service-response-examples/get-events.json [hunk 2]
```diff
@@ -0,0 +1,590 @@
+[
```
Comment (scriptist) [inline docs/service-response-examples/get-events.json:1]: Is this helpful to keep around?

**Review by scriptist — CHANGES_REQUESTED** [review]

File: docs/service-response-examples/get-events.json (hunk 2 above)
Comment (kamcknig) [reply docs/service-response-examples/get-events.json:1]: Eh, it's your repo. I can remove it if you want. But nice to have a reference without having to load the site and view the network tab

**Review by scriptist — APPROVED** [review]
Fair enough - I'm happy to leave in the json stuff if it helps :) Just trying to understand your use-case & making sure it wasn't just a mistake.

**Pull request merged**